tests/test_Account.py

```diff
@@ -47,21 +47,39 @@ async def test_execute(account_factory):
     execution_info = await initializable.initialized().call()
     assert execution_info.result == (0,)
 
-    await signer.send_transaction(account, initializable.contract_address, 'initialize', [])
+    await signer.send_transactions(account, [(initializable.contract_address, 'initialize', [])])
 
     execution_info = await initializable.initialized().call()
     assert execution_info.result == (1,)
 
+@pytest.mark.asyncio
+async def test_multicall(account_factory):
+    starknet, account = account_factory
+    initializable_1 = await starknet.deploy("contracts/Initializable.cairo")
+    initializable_2 = await starknet.deploy("contracts/Initializable.cairo")
+
+    execution_info = await initializable_1.initialized().call()
+    assert execution_info.result == (0,)
+    execution_info = await initializable_2.initialized().call()
+    assert execution_info.result == (0,)
+
+    await signer.send_transactions(account, [(initializable_1.contract_address, 'initialize', []), (initializable_2.contract_address, 'initialize', [])])
```

Comment:
we could split this into multiple lines to make it more readable and easier to modify

Reply:
Done.